x-pack/plugins/alerting/common/routes/maintenance_window/apis/find/schemas/v1.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { maintenanceWindowResponseSchemaV1 } from '../../../response';
+
+export const findMaintenanceWindowsRequestQuerySchema = schema.object({
+  page: schema.maybe(
+    schema.number({
+      defaultValue: 1,
+      min: 1,
+      meta: {
+        description: 'The page number to return.',
+      },
+    })
+  ),
+  per_page: schema.maybe(
+    schema.number({
+      defaultValue: 10,
```

Comment:
nit: When calling the endpoint without defining `per_page`, the response seems to indicate that the default value used for the page size is 20:

<img width="164" alt="image" src="https://github.com/user-attachments/assets/96bb75df-7bbb-4558-a309-ff42e21bc259">

Reply:
Good point. Let's change the default to `20`.

Reply:
Good point!